hack/update-generated-protobuf.sh

```diff
@@ -37,7 +37,13 @@ function prereqs() {
   KUBE_BUILD_CONTAINER_NAME="kube-build-${KUBE_ROOT_HASH}"
   KUBE_BUILD_DATA_CONTAINER_NAME="kube-build-data-${KUBE_ROOT_HASH}"
   DOCKER_MOUNT_ARGS=(
-    --volume "${REPO_DIR:-${KUBE_ROOT}}:/go/src/${KUBE_GO_PACKAGE}"
+    --volume "${REPO_DIR:-${KUBE_ROOT}}/cluster:/go/src/${KUBE_GO_PACKAGE}/cluster"
```

Comment:
The latter.

The problem is basically, that we want to avoid mounting the k8s.io/kubernetes/_output directory. If there is an easier way to mount all but this one, then it would be preferred. But I don't know how to do it.